components/brave_new_tab_ui/containers/app.tsx

```diff
@@ -27,37 +27,40 @@ interface Props {
   braveTodayData: BraveTodayState
 }
 
-class DefaultPage extends React.Component<Props, {}> {
-  render () {
-    const { newTabData, braveTodayData, gridSitesData, actions } = this.props
+function dismissBraveTodayIntroCard () {
+  PreferencesAPI.saveIsBraveTodayIntroDismissed(true)
+}
 
-    // don't render if user prefers an empty page
-    if (this.props.newTabData.showEmptyPage && !this.props.newTabData.isIncognito) {
-      return <div />
-    }
+function DefaultPage (props: Props) {
+  const { newTabData, braveTodayData, gridSitesData, actions } = props
 
-    return this.props.newTabData.isIncognito
-      ? <NewPrivateTabPage newTabData={newTabData} actions={actions} />
-      : (
-        <NewTabPage
-          newTabData={newTabData}
-          todayData={braveTodayData}
-          gridSitesData={gridSitesData}
-          actions={actions}
-          saveShowBackgroundImage={PreferencesAPI.saveShowBackgroundImage}
-          saveShowStats={PreferencesAPI.saveShowStats}
-          saveShowToday={PreferencesAPI.saveShowToday}
-          saveShowRewards={PreferencesAPI.saveShowRewards}
-          saveShowTogether={PreferencesAPI.saveShowTogether}
-          saveShowBinance={PreferencesAPI.saveShowBinance}
-          saveShowAddCard={PreferencesAPI.saveShowAddCard}
-          saveShowGemini={PreferencesAPI.saveShowGemini}
-          saveShowBitcoinDotCom={PreferencesAPI.saveShowBitcoinDotCom}
-          saveShowCryptoDotCom={PreferencesAPI.saveShowCryptoDotCom}
-          saveBrandedWallpaperOptIn={PreferencesAPI.saveBrandedWallpaperOptIn}
-        />
-      )
+  // don't render if user prefers an empty page
+  if (props.newTabData.showEmptyPage && !props.newTabData.isIncognito) {
+    return <div />
```

Comment:
nit: any reason not to just return `null` here?

Reply:
not sure, I just moved this code from a class to a function (for no particular reason in the end as I didn't use). I think the div is for the blank page? But there's no styling. And in fact this pref is not real. So this code will never fire! We should probably remove that at some point then...